docs/QUICKSTART.md

```diff
@@ -0,0 +1,410 @@
+# Quick-Start Guide for uVisor on mbed OS
```

Comment:
Listing the assumptions is good.

Consider expanding `This guide will help you get started with uVisor on mbed OS.` to:
`This guide will help you get started with uVisor on mbed OS by walking you through creating a sample application for the K64F.`